cu-calendar.sty

```diff
@@ -112,13 +112,13 @@
 }%
 %
 \def\cuDayName#1{%
-  \ifcase#1 воскресение\or%
-    понедельник\or
-    вторник\or
-    среда\or
-    четверг\or
-    пятница\or
-    суббота\fi
+  \ifcase#1 недѣ́лѧ\or%
+    понедѣ́льникъ\or
+    вто́рникъ\or
+    среда̀\or
+    четверто́къ\or
+    пѧто́къ\or
+    сꙋббѡ́та\fi
 }%
```

Comment:
Agree. I will code the first one then, as its the easier one to implement and explain (in the docs).

The whole issue may be not that important because date format using DOW is probably never needed in practice. This can be improved later if we get user feedback.